Mathlib/Topology/Connected/PathConnected.lean

```diff
@@ -261,12 +272,12 @@ theorem pathComponentIn_mono {G : Set X} (h : F ⊆ G) :
 
 /-- A set `F` is path connected if it contains a point that can be joined to all other in `F`. -/
 def IsPathConnected (F : Set X) : Prop :=
-  ∃ x ∈ F, ∀ {y}, y ∈ F → JoinedIn F x y
+  ∃ x ∈ F, ∀ ⦃y⦄, y ∈ F → JoinedIn F x y
 
 theorem isPathConnected_iff_eq : IsPathConnected F ↔ ∃ x ∈ F, pathComponentIn x F = F := by
   constructor <;> rintro ⟨x, x_in, h⟩ <;> use x, x_in
   · ext y
-    exact ⟨fun hy => hy.mem.2, h⟩
+    exact ⟨fun hy => hy.mem.2, @h _⟩
```

Comment:
is this change required? (I'm just looking at the web diff - perhaps this is obvious in the editor)

Reply:
I checked myself; yes, it is.